### PR TITLE
Pal 824 test example deploy

### PR DESCRIPTION
**Pull request by dev930018**

### CODE REVIEW

File: deployment/aws-k8s/example-runner/deployServicesToK8s.sh [hunk 1]
```diff
@@ -44,7 +44,7 @@ fi
 
 # Begin script in case all parameters are correct
 helm dep up
-helm upgrade --install palisade . \
+helm upgrade --install --wait --atomic palisade . \
```
Comment (kg981167224) [inline deployment/aws-k8s/example-runner/deployServicesToK8s.sh:47]: Are we really sure we want atomic?
Problem is if it fails there is no way to get any debug or log info out. Then you have to remove the atomic and re-run hoping to get the same issue.
The argument against this is you potentially leave pods thrashing.

**Review by kg981167224 — APPROVED** [review]
Just a question on use of atomic on deploy.

File: charts/example-runner/values.yaml [hunk 2]
```diff
@@ -20,8 +20,8 @@ service:
 
 image:
   name: example-runner
-  base: jre
-  tag: SNAPSHOT-1c92114
+  base: jdk
```
Comment (developer01189998819991197253) [inline charts/example-runner/values.yaml:23]: why the change from jre to jdk?

File: deployment/aws-k8s/example-runner/deployServicesToK8s.sh (hunk 1 above)
Comment (developer01189998819991197253) [reply deployment/aws-k8s/example-runner/deployServicesToK8s.sh:47]: still outstanding

**Review by developer01189998819991197253 — COMMENTED** [review]
1 outstanding commend from @kg981167224 and one question from myself

**Review by dev-958 — APPROVED** [review]

**Review by dev-958 — APPROVED** [review]
